src/meta/processors/BaseProcessor.inl

```diff
@@ -188,6 +188,43 @@ ErrorOr<nebula::cpp2::ErrorCode, int32_t> BaseProcessor<RESP>::autoIncrementId()
 }
 
 
+template<typename RESP>
+ErrorOr<nebula::cpp2::ErrorCode, int32_t>
+BaseProcessor<RESP>::autoIncrementIdInSpace(GraphSpaceID spaceId) {
+    folly::SharedMutex::WriteHolder holder(LockUtils::localIdLock());
+    auto localIdkey = MetaServiceUtils::localIdKey(spaceId);
+    int32_t id;
+    std::string val;
+    auto ret = kvstore_->get(kDefaultSpaceId, kDefaultPartId, localIdkey, &val);
+    if (ret != nebula::cpp2::ErrorCode::SUCCEEDED) {
+        if (ret != nebula::cpp2::ErrorCode::E_KEY_NOT_FOUND) {
+            return ret;
+        }
+        id = 1;
```

Comment:
You can't set to 1. This is a bit more complicated when there is old data, you need to think about it carefully. Better add a ut to mock this case as well.

Reply:
When upgrading, when the current space has an old schema, 
you only need to find the maximum value of the tagId, edgetype, and indexid under the current space,
and the id is the maximum value +1
Then insert the key value into the meta.

This approach has changed very little.

Reply:
But this needs to be done in the meta upgrade tool.

Reply:
`maximum value of old autoIncrementId +1` sounds ok. Meta is upgraded in-place, no extra tools. If you set to 1, the old data will be messed up.

Reply:
Yes, I mean in-place updated meta.
Just deal with it here.

Reply:
In order to be compatible with the existing old schema and implementation convenience, when the local_id key of the current space does not exist, it is directly set to autoIncrementId + 1.  thx